modules/mapper-extras/src/test/java/org/elasticsearch/index/mapper/extras/RankFeatureFieldMapperTests.java

```diff
@@ -106,23 +114,131 @@ public void testDefaults() throws Exception {
         assertTrue(freq1 < freq2);
     }
 
-    public void testNegativeScoreImpact() throws Exception {
-        DocumentMapper mapper = createDocumentMapper(
-            fieldMapping(b -> b.field("type", "rank_feature").field("positive_score_impact", false))
+    /**
```

Comment:
These tests look very complex and unnecessary to my mind. Let's instead of them use something simple, similar to [KeywordFieldMapperTests::testNullValue()](https://github.com/elastic/elasticsearch/blob/main/server/src/test/java/org/elasticsearch/index/mapper/KeywordFieldMapperTests.java#L243)